### ISSUE TITLE
Outgoing shares not listed in file/folder details 

### ISSUE THREAD

**Issue by simonspa** [nextcloud/server#13690]
Since the update from Nextcloud 14.0 to 15.0, outgoing shares are not shown anymore in the "details" view. Incoming shares are still displayed as "this file has been shared with you by X". The file list correctly shows the "shared" status, but only after refreshing the page.

![image](https://user-images.githubusercontent.com/1677436/51431574-100d7680-1c2b-11e9-9ee9-7ab7d6facae3.png)

This situation leaves me without the option to unshare.

The problem appears for files, folder. It doesn't matter if it is a hsare to another user, a federated share or one by link or e-mail.

### Steps to reproduce
1. Update to NC15
2. Share a file with another user
3. Reload the page to make the "shared" next to the file appear
3. Click on "shared" and observe that only an empty side panel is shown, with the option to add further shares.

### Expected behaviour
A list of shares for this file should appear, as it did for NC14.

### Actual behaviour
Empty panel, no option to remove share.

### Server configuration

**Operating system**: Debian 

**Web server:** nginx /1.10.3

**Database:** mysql 10.1.37-MariaDB-0

**PHP version:** PHP  7.2.14

**Nextcloud version:** 15.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** Update from 14

**Where did you install Nextcloud from:** manually via zip file

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
# sudo -u www-data php ./occ app:list
Enabled:
  - accessibility: 1.1.0
  - activity: 2.8.2
  - admin_audit: 1.5.0
  - bookmarks: 0.16.0
  - bruteforcesettings: 1.3.0
  - calendar: 1.6.4
  - cloud_federation_api: 0.1.0
  - comments: 1.5.0
  - contacts: 3.0.1
  - dav: 1.8.1
  - drawio: 0.9.2
  - event_update_notification: 0.3.2
  - federatedfilesharing: 1.5.0
  - federation: 1.5.0
  - files: 1.10.0
  - files_downloadactivity: 1.4.0
  - files_external: 1.6.0
  - files_markdown: 2.0.5
  - files_pdfviewer: 1.4.0
  - files_sharing: 1.7.0
  - files_texteditor: 2.7.0
  - files_trashbin: 1.5.0
  - files_versions: 1.8.0
  - files_videoplayer: 1.4.0
  - firstrunwizard: 2.4.0
  - gallery: 18.2.0
  - impersonate: 1.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.3.0
  - metadata: 0.8.0
  - nextcloud_announcements: 1.4.0
  - notes: 2.5.1
  - notifications: 2.3.0
  - oauth2: 1.3.0
  - password_policy: 1.5.0
  - passwords: 2019.1.0
  - previewgenerator: 2.0.0
  - provisioning_api: 1.5.0
  - quota_warning: 1.4.0
  - serverinfo: 1.5.0
  - sharebymail: 1.5.0
  - socialsharing_email: 1.0.5
  - support: 1.0.0
  - systemtags: 1.5.0
  - tasks: 0.9.8
  - telephoneprovider: 1.0.2
  - theming: 1.6.0
  - twofactor_backupcodes: 1.4.1
  - updatenotification: 1.5.0
  - workflowengine: 1.5.0
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
# sudo -u www-data php occ config:list system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "15.0.2.0",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "Europe\/Berlin",
        "installed": true,
        "maintenance": false,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0
        },
        "log_rotate_size": "104857600",
        "theme": "",
        "loglevel": 2,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "htaccess.RewriteBase": "\/",
        "updater.release.channel": "stable",
        "data-fingerprint": "58897020af7835d5d5eebe9f70033c28",
        "preview_max_x": 2048,
        "preview_max_y": 2048,
        "activity_expire_days": 90,
        "knowledgebaseenabled": false,
        "simpleSignUpLink.shown": false
    }
}
```
</details>

**Are you using external storage, if yes which one:** none

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox 64, Chrome 54

**Operating system:** Ubuntu 18.10

**Comment by violoncelloCH** [comment]
thank you for reporting!
probably a duplicate of nextcloud/socialsharing#29 and / or #13521 
if not, feel free to reopen the issue

**Comment by Huster-at-github** [comment]
Same here. 
Disabling all Social Apps did not help.
Also deactivating and activating "Allow users to share by link ..." did not help.

**Comment by violoncelloCH** [comment]
@Huster-at-github do you have an adblocker active? then maybe deactivating e.g. cosmetic filters in ublock might help as stated in #13521

**Comment by Huster-at-github** [comment]
Yes, deactivating UBO worked. 
Strange ...... UBO told me it's blocking nothing:

![grafik](https://user-images.githubusercontent.com/3857906/51433678-5923f180-1c50-11e9-8ed7-f552d00d5b1d.png)

**Comment by simonspa** [comment]
Thanks @violoncelloCH - it's definitely the socialsharing_email app. Deactivating it shows the full list of active shares. I subscribed to the relevant tickets you linked.

uBlock was not to be blamed in my case.
